R/db-odbc-mssql.R

```diff
@@ -101,3 +101,37 @@
   DBI::dbExecute(con, sql)
 }
 
+#' @export
+`db_compute.Microsoft SQL Server` <- function(
+                con,
+                table,
+                sql,
+                temporary = TRUE,
+                unique_indexes = list(),
+                indexes = list(),
+                ...) {
+
+
+  # The user will have to pass the pound sign, or signs, to setup the
+  # temp table. Two pound signs indicate a global temporary table, a
+  # single pound sign indicates a local temporary table
```

Comment:
In most cases, you don't need to know the name, because you'll take the tbl directly. 

We could also display a message notifying what the user is doing.